.gitignore

```diff
@@ -8,4 +8,4 @@ docs/build/
 
 **/.DS_Store
 
-**/nitro-testnode
```

Comment:
I think `**/nitro-testnode` includes `/nitro-testnode`. Are you sure this changes anything?

I just tried creating a `nitro-testnode` directory with some files inside, and it is properly ignored 👀

Reply:
![Screenshot 2024-06-13 at 18 32 37](https://github.com/OpenZeppelin/rust-contracts-stylus/assets/22298999/3cada23d-8225-4b13-89e5-2a1d79251c3c)

Reply:
Just seemed that it changed my ide's coloured highlight for ignored folder. But yep. Same difference